ibis/expr/tests/test_table.py

```diff
@@ -1049,6 +1049,22 @@ def test_union(table):
         t1.union(t3)
 
 
+def test_intersect(table):
+    schema1 = [('key', 'string'), ('value', 'double')]
+    schema2 = [('key', 'string'), ('key2', 'string'), ('value', 'double')]
+    t1 = ibis.table(schema1, 'foo')
+    t2 = ibis.table(schema1, 'bar')
+    t3 = ibis.table(schema2, 'baz')
+
+    result = t1.intersect(t2)
+    assert isinstance(result.op(), ops.Intersection)
+
+    with pytest.raises(
+        RelationError, match=r'Table schemas must be equal for set operations'
```

Comment:
Probably not important here, but in general would be better to have one unit test for each of them.

Also, if you do that, it's a good practice to have more descriptive names, like `test_intersect_with_same_schemas_work`, `test_intersect_with_different_schemas_raises`... This way, when the test fails, you know quite well the problem without much research.

Reply:
@datapythonista This is actually the same as above, I was just following the format that union was tested in

Reply:
actually can you split these into 2 separate tests. this was just a single op before and now its 2 where we may want to have multiple tests per.